cmd/infracost/register.go

```diff
@@ -20,6 +20,25 @@ func registerCmd(ctx *config.RunContext) *cobra.Command {
 		Short: "Register for a free Infracost API key",
 		Long:  "Register for a free Infracost API key",
 		RunE: func(cmd *cobra.Command, args []string) error {
+
+			var msg string
+			var isRegenerate bool
+
+			if _, ok := ctx.Config.Credentials[ctx.Config.PricingAPIEndpoint]; ok {
+
+				isRegenerate = true
+				fmt.Printf("\nYou already have an Infracost API key saved in %s.\n", config.CredentialsFilePath())
```

Comment:
```suggestion
				fmt.Printf("You already have an Infracost API key saved in %s.\n", config.CredentialsFilePath())
```

We don't need a new line at the start here, since it's the first message that's printed

Reply:
sounds good.